core/tests/puppeteer-acceptance-tests/blog-post-draft-blogs.js

```diff
@@ -0,0 +1,33 @@
+const acceptanceTests = require("./utility-functions/puppeteer_utils.js");
+const testConstants = require("./utility-functions/testConstants.js");
```

Comment:
Why are the naming conventions for the file in the line above and this one different (lowerCamelCase vs snake_case)? We should be consistent.

Reply:
We can go with snake_case convention here